ansible_collections/arista/avd/roles/eos_l3ls_evpn/defaults/main.yml

```diff
@@ -88,6 +88,7 @@ platform_settings:
       non_mlag: 330
   - platforms: [ 7800R3, 7500R3, 7500R, 7280R3, 7280R2, 7280R ]
     tcam_profile: vxlan-routing
+    lag_hardware_only: true
```

Comment:
Changing this default should be considered a breaking change and must be flagged in release-notes. @carlbuchmann

Reply:
noted